sources/packages/backend/apps/test-db-seeding/src/db-seeding/02-institution/basic-seeding/institution-user.ts

```diff
@@ -0,0 +1,122 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import {
+  Institution,
+  InstitutionUser,
+  InstitutionUserAuth,
+  User,
+} from "@sims/sims-db";
+import {
+  InstitutionUserRoles,
+  InstitutionUserTypes,
+} from "@sims/sims-db/entities/user-types.enum";
```

Comment:
`@sims/sims-db` should be enough, if not please adjust the index on `@sims/sims-db`.